lib/logstash/inputs/stomp.rb

```diff
@@ -46,15 +46,15 @@ def connect
       @client.connect
       @logger.info("Connected to stomp server") if @client.connected?
```

Comment:
Is there a case where client.connected? can return false - if so, there is a hole in our handling of this case

Reply:
Hmmmm, this was carried over from existing code, so I honestly can't answer that. 

Given that this is an unsupported plugin I'm not inclined to investigate further immediately. I suspect its redundant, but I'd rather wait for a bug report to take action here.